flake.nix

```diff
@@ -10,6 +10,10 @@
     agenix-shell.url = "github:aciceri/agenix-shell";
     agenix-shell.inputs.nixpkgs.follows = "nixpkgs";
     devenv.url = "github:hercules-ci/devenv/flake-module";
+    clan.url = "https://git.clan.lol/clan/clan-core/archive/main.tar.gz";
```

Comment:
Does it use `export-subst`?
Probably not, but otherwise you expose yourself and users to a [reproducibility problem](https://github.com/NixOS/nix/pull/11467#pullrequestreview-2298896870).

Reply:
Our repository doesn't use that and never will.